doc/api/assert.md

```diff
@@ -208,7 +208,7 @@ changes:
     description: Enumerable symbol properties are now compared.
   - version: v9.0.0
     pr-url: https://github.com/nodejs/node/pull/15036
-    description: NaN is now compared using the [SameValueZero][] comparison.
+    description: NaN is now compared using the [SameValue Comparison][].
```

Comment:
That is actually not correct. At that point of time it was a `SameValueZero` comparison. The same applies to the other entry.

Reply:
Sorry. What link should I add to the bottom list? There is no `[SameValueZero]` there.

Reply:
The linter does not recognize the references in the comment. Therefore it would complain in case you want to add it. At least that was the case why the link was originally removed.

The best solution out of my perspective is to just remove the reference and keep the plain `SameValueZero` and maybe add backticks around it.

Reply:
Can we inline the link like `[SameValueZero](https://tc39.github.io/ecma262/#sec-samevaluezero)`?

Reply:
That should be possible as far as I can tell and that is probably even a better solution.